lib/l10n/localization.dart

```diff
@@ -13,7 +13,7 @@ Localization localization(LocalizationRef ref) {
   return switch (appLocale.languageCode) {
     'en' => _en,
     'ja' => _ja,
-    _ => _ja,
+    _ => _en,
```

Comment:
I think current code doesn't pass this path.
But considering future changes and understandable code, it's better to change this to en also.

Reply:
Great!